ingestion/src/metadata/orm_profiler/profiler/runner.py

```diff
@@ -42,24 +43,36 @@ def __init__(
         table: DeclarativeMeta,
         sample: Union[DeclarativeMeta, AliasedClass],
         partition_details: Optional[Dict] = None,
+        profile_sample_query: Optional[str] = None,
     ):
         self._session = session
         self.table = table
         self._sample = sample
         self._partition_details = partition_details
+        self._profile_sample_query = profile_sample_query
 
     def _build_query(self, *entities, **kwargs) -> Query:
         return self._session.query(*entities, **kwargs)
 
     def _select_from_sample(self, *entities, **kwargs):
         return self._build_query(*entities, **kwargs).select_from(self._sample)
 
+    def _select_from_user_query(self, *entities, **kwargs):
+        user_query = self._session.query(self.table).from_statement(
+            text(f"""{self._profile_sample_query}""")
```

Comment:
do we need all the """""""?

Reply:
Regular quoting should be ok. Just wanted to be extra safe but looks like case such as `SELECT "id" FROM "table"` are managed correctly.